src/docbrowser/documentation.ts

```diff
@@ -137,15 +138,8 @@ async function getDocumentation(): Promise<string> {
     )
 }
 
-function setDocumentation(inner: string) {
-    if (!inner) { return }
-    g_forwardStack = [] // initialize forward page stack for manual search
-    showDocumentationPane()
-    const html = createWebviewHTML(inner)
-    _setHTML(html)
-}
-
-function createWebviewHTML(inner: string) {
+function createWebviewHTML(docAsMD: string) {
+    const docAsHTML = md.render(docAsMD)
```

Comment:
Do we want to wrap this in try/catch ?

Reply:
I don't think so, if there is a bug in `md.render` we would want to know about it, right? Or could this throw an error because of a malformed doc string? Is there such a thing as invalid markdown?